docs/testing/releases/740.md

```diff
@@ -0,0 +1,86 @@
+
+### Fix Featured Product: Prevent the block from breaking for out of stock products hidden from catalog. ([6166](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/6166))
+
+1. Add a Featured Product block to a page and save
+2. Head to the product and set the product to Out of Stock
+3. Under WooCommerce > Settings > Products > Inventory, check the box that says "Hide out of stock items from the catalog"
+4. Return to the page with your Featured Product block, select it, and select "Edit" to choose a new product to feature
+5. Make sure the block can still be edited to choose a new product and the Out of Stock product is also available.
+
+### Mini Cart block: Fix translations loading. ([6158](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/6158))
+
+1. Go to **Settings -> General** and change the language of the site to any language other than English (my suggestion is to use Deutsch because all the translations are available).
+2. Enable a classic theme (for example Storefront)
+3. Now go to **Appearance -> Widgets** and add `Mini Cart` widget to any Widget Area.
+4. Go to any page containing that Widget Area and check that any error related to `wp.i18n`.
+5. Check that the `Mini Cart` is translated (check with an empty cart and with a filled cart). Please, be sure that you are using a language that has translations for the Mini Cart Block. My suggestion is to use the Deutsch language.
+6. Enable a block theme (for example TT2).
+7. Now go to **Appearance -> Editor** and add the `Mini Cart` block in the header. Save.
+8. Check that the `Mini Cart` is translated (check with an empty cart and with a filled cart). Please, be sure that you are using a language that has translations for the Mini Cart Block. My suggestion is to use the Deutsch language.
+
+### Fix Featured Product and Featured Category buttons misalignment in Twenty Twenty Two theme. ([6156](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/6156))
+
+1. Activate a block theme (ie: Twenty Twenty-Two).
+2. Add Featured Product and/or Featured Category blocks to a post or page.
+3. Verify the button is centered by default.
+4. Verify you can change the alignment of the button using the _Justify items_ dropdown inherited from the _Buttons_ block.
+![imatge](https://user-images.githubusercontent.com/3616980/160625173-f9ad42ed-b769-42e3-9ad8-3f3abe60b61c.png)
+5. Test again with a classic theme (ie: Storefront).
+
+### Remove the ToggleButtonControl in favor of ToggleGroupControl. ([5967](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/5967))
+
+1. Check out this branch and compile the changes
+2. Activate a **block theme**, like Twenty Twenty Two
+3. Create a new page, and add the following blocks:
+   - Active Filters
+   - Attribute Filter
+   - Price Filter
+   - Product Categories
+   - Reviews
+4. Ensure that options are using the new component and are working correctly
```

Comment:
Please include the screenshots from #5967 in the testing instructions - the text instructions on their own are unclear and the screenshots it will make it much easier to test.

Reply:
Screenshots added!